releasenotes/notes/mcx_create_explicit_c3x_c4x-d4475c0bb70fae49.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
```

Comment:
I think this should probably be an upgrade note, not a feature. It's a change in behavior from earlier even if the output is more efficient.

Reply:
won't changes to backports create merge headaches?

Reply:
It shouldn't (because we don't ever merge the stable branches back into master).

Reply:
```suggestion
upgrade:
```